source/server.ts

```diff
@@ -54,6 +54,18 @@ export function createServer(options = {} as ServerOptions): Server {
     uiManager.drawRequest(req, res, next)
   );
 
+  expressServer.use((req: Request, res, next) => {
+    const route = routeManager.findRouteByPath(req.path.replace(basePath, ''));
+    if (route) {
+      const proxy = proxyManager.resolveRouteProxy(route);
+      if (proxy) {
+        return proxy?.(req, res, next);
```

Comment:
You've already checked if proxy does exist, is `proxy?.` really necessary in here?

Reply:
Nope, but in the next upcoming pull request I fixed this.